src/Interface/Modules/Render/ViewScene.h

```diff
@@ -39,6 +39,7 @@ DEALINGS IN THE SOFTWARE.
 #include <Interface/Modules/Render/ViewSceneControlsDock.h>
 #include <Graphics/Datatypes/GeometryImpl.h>
 #include <Interface/Modules/Render/share.h>
+#include <glm/gtc/quaternion.hpp>
```

Comment:
Part of: Move this function to a new header/cc file, as a global function or in a new class, called ViewSceneUtility or the like.

Two problems here: the glm is after the share.h. Also, we want to minimize glm includes in our header code.